Sources/REventLogger/Networking/ConfigureUrlRequest.swift

```diff
@@ -0,0 +1,71 @@
+import Foundation
+
+protocol ConfigureUrlRequest {
+    var method: HTTPMethod { get }
+    var header: [String: String]? { get }
+    var path: String { get }
+    func createURLRequest(with url: URL, body: [REvent]?) -> URLRequest?
```

Comment:
we can add body as encodable variable like below so it would be more generic. 
`var body: Encodable? { get }`

Reply:
@Esakkiraja-Pothikannan will there be any decoding of `REvent`?

Reply:
Indention is to keep as encodable so it can be encoded at one place before create request.